client/src/components/InfoAndLinks.js

```diff
@@ -3,8 +3,9 @@ import RecyclingInfo from '../components/RecyclingInfo';
 import PropManagerLinks from '../components/PropManagerLinks';
 
 const InfoAndLinks = () => (
-  <div>
+  <div className="info-and-links">
```

Comment:
** [class-wrapper-names] ** We have been trying to use a wrapper syntax for these types of classes. Should we consider that here too?  I know the app has lots of inconsistencies. Thoughts?

Reply:
Sure, I think it would be great to keep that consistent.